packs/data/feats.db/mountain-quake.json

```diff
@@ -13,7 +13,7 @@
             "value": 1
         },
         "description": {
-            "value": "<p><strong>Trigger</strong> You are in @Compendium[pf2e.feats-srd.Mountain Stance]{Mountain Stance}</p>\n<hr />\n<p>You stomp, shaking the earth beneath you. Creatures on the ground within a @Template[type:emanation|distance:20] take damage equal to your Strength modifier (minimum 0), which they can resist with a basic Fortitude save against your class DC. On a failure, they also fall @Compendium[pf2e.conditionitems.Prone]{Prone}.</p>\n<p>After you use this action, you can't use it again for [[/r 1d4 #rounds]]{1d4 rounds}.</p>\n<hr />\n<p><strong>Special</strong> If you have this feat, the Dexterity modifier cap to your AC while using Mountain Stance increases from +1 to +2.</p>"
+            "value": "<p><strong>Requirements</strong> You are in @Compendium[pf2e.feats-srd.Mountain Stance]{Mountain Stance}</p>\n<hr />\n<p>You stomp, shaking the earth beneath you. Creatures on the ground within a @Template[type:emanation|distance:20] take damage equal to your Strength modifier (minimum 0), which they can resist with a @Check[type:fortitude|dc:resolve(@actor.system.proficiencies.classDCs.monk.value)|basic:true] save against your class DC. On a failure, they also fall @Compendium[pf2e.conditionitems.Prone]{Prone}.</p>\n<p>After you use this action, you can't use it again for [[/r 1d4 #rounds]]{1d4 rounds}.</p>\n<hr />\n<p><strong>Special</strong> If you have this feat, the Dexterity modifier cap to your AC while using Mountain Stance increases from +1 to +2.</p>"
```

Comment:
This should grab the highest DC (`@actor.system.attributes.classDC.value`) since it can be taken via the Martial Artist archetype.